test/e2e/kubectl.go

```diff
@@ -473,6 +474,25 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 		})
 	})
 
+	framework.KubeDescribe("Kubectl should be able to talk to api server", func() {
+		It("kubectl runing in a pod should talk to api server [Conformance]", func() {
```

Comment:
In the exec/http proxy e2e test, we used to find kubectl from the release tarball or build one locally, and once we had it, we uploaded it to a netexec pod, and then executed it remotely. We removed that complex logic and simplified it greatly, so we're not doing that any more. There may be other tests where we run kubectl in a pod, but I am not familiar with them off the top of my head.